dev-client/src/components/SiteChip.tsx

```diff
@@ -15,12 +15,12 @@
  * along with this program. If not, see https://www.gnu.org/licenses/.
  */
 
-import {ChipBadge} from 'terraso-mobile-client/components/ChipBadge';
+import {Chip} from 'terraso-mobile-client/components/Chip';
 
 type Props = {
   count: number;
 };
 
-export const PeopleBadge = ({count}: Props) => {
-  return <ChipBadge count={count} iconName="people-alt" />;
+export const SiteChip = ({count}: Props) => {
```

Comment:
I think that, given 3-4 types of `Chip` (the base, the site one, the people one, and the role one) we should have a `chips` subdirectory under `components`. I would also circle back to Courtney to let her know that we have these dedicated `Chip` variants so she can document their existence in the component library.